tests/pillar/network.sls

```diff
@@ -88,3 +88,29 @@ linux:
       #     initial_interval: 12
       #     reject:
       #       - 10.0.4.0/24
+    systemd:
+      link:
+        10-iface-dmz:
+          Match:
```

Comment:
Uppercase characters is not standard that we use, you should use lower-case instead - metadata should drive states and configs not the other way round.